lnbits/wallets/fake.py

```diff
@@ -17,13 +17,9 @@
     Wallet,
 )
 
-env = Env()
-env.read_env()
-
 
 class FakeWallet(Wallet):
-    queue: asyncio.Queue = asyncio.Queue(0)
-    secret: str = env.str("FAKE_WALLET_SECTRET", default="ToTheMoon1")
+    secret: str = settings.fake_wallet_secret
```

Comment:
- `queue: asyncio.Queue = asyncio.Queue(0)` has been moved to `paid_invoices_stream()`
- but `self.queue` is also used in `pay_invoice()`